README.md

```diff
@@ -26,23 +26,23 @@ This repository can be used to deploy OpenWhisk to a Kubernetes cluster.
 
 # Table of Contents
 
-* [Requirements](#requirements)
-* [Setting up Kubernetes](#setting-up-kubernetes)
-* [Configuring OpenWhisk](#configuring-openwhisk)
+* [Setting up Kubernetes and Helm](#setting-up-kubernetes-and-helm)
+* [Deploying OpenWhisk](#deploying-openwhisk)
 * [Cleanup](#cleanup)
 * [Issues](#issues)
 
-# Requirements
-Several requirements must be met for OpenWhisk to deploy on Kubernetes.
+# Setting up Kubernetes and Helm
+
+## Kubernetes
+
+### Requirements
 
-**Kubernetes**
-* [Kubernetes](https://github.com/kubernetes/kubernetes) version 1.6+. However, multiple minor releases of Kubernetes, including 1.6.3, 1.7.14, 1.8.9 and 1.9.4 will not work for OpenWhisk due to bugs with volume mount subpaths (see[[1](https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG-1.6.md#known-issues-for-v163), [2](https://github.com/kubernetes/kubernetes/issues/61076)]. This bug will surface as a failure when deploying the nginx container.
+Several requirements must be met for OpenWhisk to deploy on Kubernetes.
+* [Kubernetes](https://github.com/kubernetes/kubernetes) version 1.7+. However, multiple minor releases of Kubernetes, including 1.7.14, 1.8.9 and 1.9.4 will not work for OpenWhisk due to bugs with volume mount subpaths (see[[1](https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG-1.6.md#known-issues-for-v163), [2](https://github.com/kubernetes/kubernetes/issues/61076)]. This bug will surface as a failure when deploying the nginx container.
```

Comment:
We don't even test version 1.7, no way to know if it works or not.

I would suggest to put here the minimum version we know it work, still point out the problems with volume mount paths.

What about 

* [Kubernetes](https://github.com/kubernetes/kubernetes) version 1.8.10+, 1.9.5+, and 1.10.x
The following releases of Kubernetes, including 1.8.9 and 1.9.4 will not work for OpenWhisk due to bugs with volume mount subpaths (see[[1](https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG-1.6.md#known-issues-for-v163), [2](https://github.com/kubernetes/kubernetes/issues/61076)]. This bug will surface as a failure when deploying the nginx container.

Reply:
Good point.  I dropped 1.7 and made some of the other adjustment.  I need to look into what minor versions of 1.8.x, 1.9.x, and 1.10.x we should recommend.